go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/IBM/keyprotect-go-client
 
-go 1.12
+go 1.15
```

Comment:
We should move to go 1.17 latest one.

Reply:
I don't believe this has a functional effect at all, only the dep versions below do